src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/ParamsCalculator.java

```diff
@@ -17,11 +18,14 @@
     @Inject
     private Environment env;
 
+    @Inject
+    private AnalysisService analysisService;
+
     private static Integer calculateHypervisors(Object e, String cpuTotalCoresPath, String cpuCoresPerSocketPath) {
         Map mapa = (Map) e;
         Integer cputotalcores = (Integer) mapa.get(cpuTotalCoresPath);
         Integer cpucorespersocket = (Integer) mapa.get(cpuCoresPerSocketPath);
-        return cputotalcores / (cpucorespersocket * 2);
+        return Double.valueOf(Math.ceil(cputotalcores / (cpucorespersocket * 2.0))).intValue();
```

Comment:
The boxing to Double and the `intValue()` could be avoided casting to `int`? 
something like `return (int) Math.ceil(cputotalcores / (cpucorespersocket * 2.0));`should work

Reply:
+1 Good catch